src/main/java/subway/domain/section/Sections.java

```diff
@@ -0,0 +1,65 @@
+package subway.domain.section;
+
+import subway.common.exception.InsufficientStationsException;
+import subway.common.exception.InvalidDownStationException;
+import subway.common.exception.InvalidUpSationException;
+import subway.common.exception.NotLastStationException;
+import subway.common.response.ErrorCode;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Embeddable
+public class Sections {
+
+    @OneToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL, orphanRemoval = true)
+    @JoinColumn(name = "line_id")
+    private List<Section> sections = new ArrayList<>();
```

Comment:
> 그리고 구현하고나서 궁금한점이..deleteSection 실행후 아래 캡쳐처럼 update가 실행되는데 괜찮은걸까요...? jpa를 공부하면서 하고있긴한데 너무 헷갈리네요 ㅠㅠ 전체적으로 코드 봐주시고 답변부탁드립니다!

질문하신 내용의 답변을 여기서 드리면 좋을 것 같습니다.
```suggestion
    @OneToMany(mappedBy = "line", fetch = FetchType.EAGER, cascade = CascadeType.ALL, orphanRemoval = true)
    private List<Section> sections = new ArrayList<>();
```
로 수정하고 Section.java 에
```
@ManyToOne(fetch = FetchType.LAZY)
@JoinColumn(name = "line_id")
private Line line;

```

이렇게 추가해주시면 됩니다 😄 
설명을 길게 적다가 .. 조금 더 정리되어있는 글을 보고 학습해보시는 것이 좋을 것 같아서, 아래 남겨드린 참고 글을 학습해보시고 추가로 궁금한 점이 있다면 말씀해주세요 🔥 

[참고](https://ksh-coding.tistory.com/105)

Reply:
[JPA 는 김영한님의 책](https://www.yes24.com/Product/Goods/19040233) 이 가장 인기있는 것으로 알고 있습니다. 사용은 안하더라도 학습은 해보시면 좋다고 생각해요 😄 
JPA 라는 기술이 나오기 까지의 과정이 많은 개발자들이 저장소와 애플리케이션을 어떻게 격리할까에 대한 깊은 고민이 담긴 것이라서요
추가로 [마틴파울러의 엔터프라이즈 아키텍처 패턴](https://www.yes24.com/Product/Goods/22384677) 이라는 책의 아이디어가 많은 ORM 기법에 녹아져 있는 것을 보면 재밌게 학습하실 수 있을 것 같아요 👍